Project/dataset target: convert note into a warning (#2284)
openmicroscopy/ome-documentation: omero/users/cli/import-target.rst

Dataset ``New Dataset`` already exists. Then, the existing Dataset will be used
for the import of the image and linked to Project ``Proj1``.

.. warning::

>    In the case where ``New Dataset`` exists and has been linked prior to your

Reviewer: ```suggestion
  If a Dataset named ``New Dataset`` already exists and has been linked prior to your
```

>    import to some other Project (for example ``ProjP``), the ``New Dataset`` will

Reviewer: ```suggestion
   import to some other Project (for example ``ProjP``), this existing Dataset will
```

>    be used as the target dataset container and will be linked  both to ``ProjP``

Reviewer: ```suggestion
   be used as the target Dataset container and will be linked  both to ``ProjP``
```
